derive/src/to_client.rs

```diff
@@ -97,9 +97,7 @@ fn generate_client_methods(methods: &[MethodRegistration]) -> Result<Vec<syn::Im
 				let arg_names = compute_arg_identifiers(&args)?;
 				let returns = match &method.attr.kind {
 					AttributeKind::Rpc { returns, .. } => compute_returns(&method.trait_item, returns)?,
-					AttributeKind::PubSub { .. } => {
-						continue;
-					}
+					_ => continue,
```

Comment:
I'd prefer to list the variants here, since modifying `AttributeKind` in the future will force handling the case here.

Reply:
Will do, once I have an answer for https://github.com/paritytech/jsonrpc/pull/454#discussion_r304851662.

Reply:
I've put the explicit value here, and I'm currently not generating anything for notifications in the client code. If we need support for this, please respond to https://github.com/paritytech/jsonrpc/pull/454#discussion_r304851662 and I can add it to this PR.